README.md

```diff
@@ -74,7 +74,9 @@ Response:
   "id": "PROCESSING_PROGRESS",
   "payload": {
     "progress": 12,
-    "total": 674
+    "total": 674,
+    "path": "/Some/user/path/to/single_photo_with_correct_path_and_name.jpeg",
+    "originalPath": "/Some/user/path/to/single_photo.jpeg"
```

Comment:
hmm, dla mnie to jest bardziej kwestia "input.path" i "output.path", zostawiasz sobie w payload fajne miejsce na metadane i lepiej definijesz czym jest, który path. Chyba że zamiast path, to jakieś target, i zamiast original coś w stylu source.path

Reply:
Też myślałem nad `input.path` i `output.path`. Może faktycznie warto zostawić sobie możliwość rozwinięcia tego modelu.

Reply:
nawet jeżeli nie bezpośrednio rozwinięcie, to nazwy path i originalPath wydają mi się kiepsko opisywać dane poprostu

Reply:
a fajniej byłoby też przy przetwarzaniu widać, który path staje się którym :)

Reply:
Fixed :)